test/test_browser.py

```diff
@@ -2555,6 +2555,8 @@ def test_worker_api_with_pthread_compilation_fails(self):
 
   @also_with_wasmfs
   def test_emscripten_async_wget2(self):
+    if self.get_setting('WASMFS'):
+      self.set_setting('FORCE_FILESYSTEM')
```

Comment:
Why is this change needed?

Reply:
I think when I initially opened this PR this test was saying `FS is not defined`, though now I suspect that error had more to do with how I was running the tests locally instead of problems with the test itself. It should be fixed now.